apps/lib/apps.c

```diff
@@ -2993,18 +2993,6 @@ void make_uppercase(char *string)
         string[i] = toupper((unsigned char)string[i]);
 }
 
-/* This function is defined here due to visibility of bio_err */
```

Comment:
IF EXIST test\aesgcmtest.exe.manifest DEL /F /Q test\aesgcmtest.exe.manifest
	"link" /nologo /debug setargv.obj /subsystem:console /opt:ref  /nologo /debug 
Windows build doesnt like this change..

libtestutil.lib(libtestutil-lib-options.obj) : error LNK2005: opt_printf_stderr already defined in libapps.lib(libapps-lib-opt_err.obj)
test\aesgcmtest.exe : fatal error LNK1169: one or more multiply defined symbols found
NMAKE :

Reply:
Yeah, I know.  I'm trying to untangle that, but that's a matter of experimentation...